content/knowledge_base/glossary.md

```diff
@@ -0,0 +1,141 @@
+# Glossary
+
+## General Blockchain Concepts
+
+### Consensus
+
+TODO: add
+
+### Blockchain
+
+TODO: add
+
+### Gossip Algorithms
+
+TODO: add
+
+### Liveness Threshold
+
+The number of malicious participants that can be tolerated before the consensus protocol stops functioning.
+
+### Nakamoto Type of Consensus
+
+TODO: introduce (for BTC)
+A consensus algorithm that provides a probabilistic safety guarantee; its decisions may revet with some probability ε.
+A permissionless protocol allowing each node to join or leave the system at any time.
+TODO: finish
+
+### Safety
+
+TODO: add
+
+## Avalanche Concepts
+
+### Ancestor Set
+
+TODO: add
+
+### Chit
+
+A transaction receives a chit (boolean value) when the majority (quorum) of queried nodes respond with an approval for that transaction.
+
+### Confidence
+
+The sum of transaction/vertex's decendants cihts and its own chit.
+The chit value is a result of a one-time query for its associated transaction and becomes immutable afterwards.
+Because chit values can be 0 or 1 - c ∈ {0,1} confidence values are monotonic.
```

Comment:
So, true or false? Or do we use 0/1 because it is used in mathematical operations?

```suggestion
Because chit values can be `0` or `1` — `c ∈ {0,1}` confidence values are monotonic.
```

Reply:
It is used to sum the chits of multiple transactions in the grapth path
Corrected the quotation here https://github.com/optakt/optakt.github.io/pull/13/commits/48a7f92c1e451ec49d7b7f2cb79fa55a85a83a48#diff-55a59bbede67c988e24e88ffb8a7b56d20454b9ab95a3fd8567e790585338dfbR88
Thanks!